frontend-sb/src/components/Search.tsx

```diff
@@ -34,12 +34,15 @@ export const Search: React.FC<Props> = (props) => {
     debounceSearch(e);
   };
 
-  const handleClickURIs = useCallback((e) => {
-    e.preventDefault();
-    const { uris } = e.currentTarget.dataset;
-    setURIs(uris.split(','));
-    setIsPlaying(true);
-  }, []);
+  const handleClickURIs = useCallback(
+    (e) => {
+      e.preventDefault();
+      const { uris } = e.currentTarget.dataset;
+      emitSearchedURIs(uris.split(','));
```

Comment:
What does this method do? Is it possible for a situation to occur where an URI has a comma and this would incorrectly split it?

Reply:
A Spotify URI has a clear spec definition:

`spotify:track:someRandomUriString` or `spotify:album:someRandomUriString` or `spotify:artist:someRandomUriString`

We shouldn't be getting any weird chars or escape strings from their API. That would be a breaking change on their side.